XamlControlsGallery/ControlPages/InfoBadgePage.xaml

```diff
@@ -0,0 +1,156 @@
+<!--
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+-->
+<Page
+    x:Class="AppUIBasics.ControlPages.InfoBadgePage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AppUIBasics"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:contract5Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 5)"
+    mc:Ignorable="d">
+
+    <StackPanel>
+        <local:ControlExample x:Name="Example1" HeaderText="InfoBadge embedded in NavigationView" HorizontalContentAlignment="Stretch" >
+            <local:ControlExample.Example>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto"/>
+                    </Grid.RowDefinitions>
+
+                    <muxc:NavigationView x:Name="nvSample1" Height="300" PaneDisplayMode="Left">
+                        <muxc:NavigationView.MenuItems>
+                            <muxc:NavigationViewItem Content="Home" Icon="Home"/>
+                            <muxc:NavigationViewItem Content="Account" Icon="Contact"/>
+                            <muxc:NavigationViewItem x:Name="InboxPage" Content="Inbox" Icon="Mail">
+                                <muxc:NavigationViewItem.InfoBadge>
+                                    <muxc:InfoBadge x:Name="infoBadge1" Value="5" Opacity="{x:Bind InfoBadgeOpacity, Mode=OneWay}" />
+                                </muxc:NavigationViewItem.InfoBadge>
+                            </muxc:NavigationViewItem>
+                        </muxc:NavigationView.MenuItems>
+                        <Frame x:Name="contentFrame" />
+                    </muxc:NavigationView>
+                </Grid>
+                
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <StackPanel Width="130">
```

Comment:
I think we should increase this, the LeftExpanded value is being truncated (at least on my machine).